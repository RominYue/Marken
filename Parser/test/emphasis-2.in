This is *a multi
line* emphasis.

This is *a
multi
line* emphasis.