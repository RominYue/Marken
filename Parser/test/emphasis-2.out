<p>This is <em>a multi
line</em> emphasis.</p>

<p>This is <em>a
multi
line</em> emphasis.</p>
